.eslintrc.json

```diff
@@ -70,7 +70,7 @@
   },
   "plugins": ["react", "no-only-tests", "react-hooks", "testing-library", "jest", "no-storage"],
   "rules": {
-    "linebreak-style": ["error", "unix"],
+    "linebreak-style": ["error", "windows"],
```

Comment:
since I'm developing in windows, had to change this to avoid eslint errors on linebreak styles